src/detection_utils/metrics.py

```diff
@@ -76,23 +74,25 @@ def compute_precision(
         return 1  # (0 TP) / (0 TP + 0 FP) is counted as 100% correct
 
     if truths.sum() == 0:
-        return 0  # we've already handled the case where we found 0/0 relevant objects above
+        return (
+            0
+        )  # we've already handled the case where we found 0/0 relevant objects above
 
     ious = box_overlaps(prediction_detections[:, :4], truth_detections[:, :4])
     max_ious = ious.max(axis=1)
     max_idxs = ious.argmax(axis=1)
 
     target_labels = truths[max_idxs]
-    true_positives = (predictions == target_labels)[np.logical_and(max_ious >= threshold, target_labels > 0)].sum()
+    true_positives = (predictions == target_labels)[
+        np.logical_and(max_ious >= threshold, target_labels > 0)
+    ].sum()
     predicted_positives = (predictions > 0).sum()
 
     return true_positives / predicted_positives
 
 
 def compute_recall(
-        prediction_detections: ndarray,
-        truth_detections: ndarray,
-        threshold: float = 0.5,
+    prediction_detections: ndarray, truth_detections: ndarray, threshold: float = 0.5
```

Comment:
same as above, just because this can fit on one line doesn't mean it should. I think it looks better/is easier to read separated